linkerd/docs/routers.md

```diff
@@ -254,6 +254,11 @@ requeueBudget | see [retry budget](#retry-budget-parameters) | A [requeue budget
 failureAccrual | 5 consecutive failures | a [failure accrual policy](#failure-accrual) for all clients created by this router.
 requestAttemptTimeoutMs | no timeout | The timeout, in milliseconds, for each attempt (original or retry) of the request made by this client.
 
+<aside class="notice">
```

Comment:
should this notice be in the http and h2 docs instead of here?

Reply:
ahh yes, I can fix that